src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EcDsa.ImportExport.cs

```diff
@@ -41,17 +42,17 @@ internal static partial class Crypto
         [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_EcKeyCreateByExplicitParameters")]
         internal static extern SafeEcKeyHandle EcKeyCreateByExplicitParameters(
             ECCurve.ECCurveType curveType,
-            byte[] qx, int qxLength,
-            byte[] qy, int qyLength,
-            byte[] d, int dLength,
+            byte[]? qx, int qxLength,
+            byte[]? qy, int qyLength,
+            byte[]? d, int dLength,
             byte[] p, int pLength,
             byte[] a, int aLength,
             byte[] b, int bLength,
             byte[] gx, int gxLength,
             byte[] gy, int gyLength,
             byte[] order, int nLength,
-            byte[] cofactor, int cofactorLength,
-            byte[] seed, int seedLength);
+            byte[]? cofactor, int cofactorLength,
+            byte[]? seed, int seedLength);
```

Comment:
In all of these cases, I'm unclear why some of the byte[] arguments are nullable and others aren't.  @bartonjs, does that logically make sense given the nature of these methods?  If yes, ok.  If no, we should be consistent and annotate these appropriately.

Reply:
Yeah, they all seemed correct when I did my pass.  The nullable things were optional or otherwise contextual.

Reply:
Ok